src/ZeroCrossImpl.java

```diff
@@ -0,0 +1,177 @@
+import java.util.Scanner;
+
+public class ZeroCrossImpl implements ZeroCross {
+
+    int whoPlays;
+    public int firstPlayer;
+    public int secondPlayer;
+
+
+    @Override
+    public void run() {
+
+
+        Scanner scanner = new Scanner(System.in);
+
+        int[][] zeroCross = new int[3][3];
+
+        System.out.println("Введите целые числа которыми будете играть в игру Крестик-Нолик" + " " + "роль нолика занимает число" +
+                "-" + "x");
+
+        int x = scanner.nextInt();
+        firstPlayer = x;
```

Comment:
Use full name of variable that can reveal meaning.
"The name of a variable, function, or class, should answer all the big questions. It
should tell you why it exists, what it does, and how it is used. If a name requires a com-
ment, then the name does not reveal its intent."